net-news/yencode/yencode-0.46-r1.ebuild

```diff
@@ -12,7 +14,7 @@ IUSE="nls"
 
 DEPEND=""
 
-src_compile() {
+src_configure() {
 	local myconf
 	use nls || myconf="--disable-nls"
 
```

Comment:
1. paths need to be prefixed with `"${EPREFIX}"`
2. remove `emake` from `src_configure`
3. `make` -> `emake` in src_install
4. `einstalldocs` instead of `dodoc ...`

Reply:
responded below :-/